### PR TITLE
better ecoscore details display

### PR DESCRIPTION
**Pull request by stephanegigandet**
Added a little bit of structure to the details of the computation.

There will probably be more to come, e.g. some icons, probably some details about the LCA impacts of ingredients, processing, transportation etc.

image

### CODE REVIEW

File: templates/ecoscore_details.tt.html [hunk 1]
```diff
@@ -43,19 +56,34 @@ <h4>Origine des ingrédients</h4>
 		<strong>Bonus / malus - Politique environnementale : [% round(adjustments.origins_of_ingredients.epi_value) %]</strong><br>
```
Comment (CharlesNepote) [inline templates/ecoscore_details.tt.html:56]: "Bonus / malus - Politique environnementale : -5" is hard to read. I had to understand that the first "-" is not a "minus" but just a "connection" sign. I would write a ">" instead or event nothing:
=> "Bonus / malus > Politique environnementale : -5"

File: templates/ecoscore_details.tt.html [hunk 2]
```diff
@@ -43,19 +56,34 @@ <h4>Origine des ingrédients</h4>
...
 		<strong>Bonus / malus - Transport : [% round(adjustments.origins_of_ingredients.transportation_value) %]</strong><br>
```
Comment (CharlesNepote) [inline templates/ecoscore_details.tt.html:57]: Same comments than "Bonus / malus - Politique environnementale".

**Review by teolemon — APPROVED** [review]

**Comment by stephanegigandet** [comment]
@CharlesNepote : good point, I'm removing the words Bonus / Malus, it's shorter and clearer that way:

image

**Review by teolemon — APPROVED** [review]
I think the footprint icon is going to create confusion among users. I would put something related to deforestation instead.

**Pull request merged**